apps/desktop/src/main/menu/menu.help.ts

```diff
@@ -166,6 +166,7 @@ export class HelpMenu implements IMenubarMenu {
       {
         id: "android",
         label: "Android",
+        visible: !isMacAppStore(),
```

Comment:
Should we add a comment here mentioning why it's hidden?

```suggestion
        visible: !isMacAppStore(), // Apple Guideline 2.3.10 - Accurate Metadata
```

Reply:
As Apple's [guidelines](https://developer.apple.com/app-store/review/guidelines/#:~:text=2.3.10%20Make%20sure%20your,is%20specific%2C%20approved%20interactive%20functionality.) are public, good idea.